src/main/java/com/future/function/qa/util/Path.java

```diff
@@ -19,4 +21,6 @@
   String COPY = "/copy";
 
   String ASSIGNMENT = "/api/scoring/batches/%s/assignments";
+
+  String ROOM = "/api/scoring/batches/batchId/assignments/%s/room/%s";
```

Comment:
Please check consistency with #99, whichever style is used is no problem, just a matter of consistency